lib/perl/Genome/Ptero/ToJSONForProcess.t

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env genome-perl
+
+use strict;
+use warnings FATAL => 'all';
+
+use Test::More;
+use above 'Genome';
+use Genome::Utility::Test qw(compare_ok);
+use File::Basename qw(dirname basename);
+use File::Spec qw();
+use File::Copy qw(copy);
+
+BEGIN {
+    $ENV{UR_DBI_NO_COMMIT} = 1;
+    $ENV{UR_USE_DUMMY_AUTOGENERATED_IDS} = 1;
+};
+
+
+for my $test_directory (glob test_data_directory('*')) {
+    my $test_name = basename($test_directory);
+    my $workflow = Genome::WorkflowBuilder::DAG->from_xml_filename(workflow_xml_file($test_name));
+
+    my $json_filename = do {
+        local %ENV; # otherwise the JSON contains the current user's environment.
```

Comment:
Would it be worthwhile to include a small environment here that includes a defined and an undefined variable?